assets/src/scss/new-identity/style.scss

```diff
@@ -42,4 +42,14 @@
   --button-secondary--hover--background: var(--p4-dark-green-800);
   --button-secondary--hover--border-color: var(--p4-dark-green-800);
   --button-secondary--hover--color: var(--white);
+  --block-spreadsheet--color: var(--grey-900);
+  --block-spreadsheet-grey-header--background: var(--grey-800);
```

Comment:
The grey colors are actually the same as the old ones, but I still added them here to use the tokens